src/components/DetailCard/DetailCard.jsx

```diff
@@ -9,7 +9,7 @@ const DetailCard = ({
   price,
   imageLink
 } = {}) => (!title ? (
```

Comment:
you could remove the ! in front of title and swap the positions of the following 2 options.

Reply:
good idea!